rest-api/api/participant_counts_over_time_api.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+# import json
+# import logging
+
+from flask.ext.restful import Resource
+from flask import request
+from werkzeug.exceptions import BadRequest
+
+from api_util import HEALTHPRO
+from api_util import get_awardee_id_from_name
+from app_util import auth_required
+from dao.hpo_dao import HPODao
+from participant_enums import EnrollmentStatus
+
+DATE_FORMAT = '%Y-%m-%d'
+DAYS_LIMIT = 100  # provisional, per design doc
+
+
+class ParticipantCountsOverTimeApi(Resource):
+
+  @auth_required(HEALTHPRO)
+  def get(self):
+    self.hpo_dao = HPODao()
+
+    # TODO: After enrollment status is filterable,
+    # wire in 'organization', 'site', and 'enrollmentStatus'.
```

Comment:
Other params in the doc not mentioned here: withdrawalStatus, bucketSize. Do you want to add those?

Reply:
I meant "withdrawalStatus" where I where "enrollmentStatus", and you're right about bucketSize.  Added in comment.